src/metabase/lib/order_by.cljc

```diff
@@ -58,10 +58,10 @@
     x
     direction    :- [:maybe [:enum :asc :desc]]]
    (let [stage-number (or stage-number -1)
-         order-by     (cond-> (->order-by-clause query stage-number x)
+         new-order-by (cond-> (->order-by-clause query stage-number x)
```

Comment:
Yeah, IMO `order-by'` wouldn't have made sense here because there wasn't an `order-by`-not-prime in the first place. We are building a new `:order-by` clause, so that's what drove this choice here.